core/src/test/java/org/apache/calcite/examples/RelBuilderExample.java

```diff
@@ -66,7 +66,7 @@ private RelBuilder doExample(RelBuilder builder, int i) {
     case 4:
       return example4(builder);
     default:
-      throw new AssertionError("unknown example " + i);
+      throw new AssertionError("unknown example" + i);
```

Comment:
why is this more accurate?

Reply:
Because it's the same way it was written, like i is 5 for example5, but test is all example4, and example5 has an extra space, so I've removed the space and turned it into example5